float64 robot_vel
float64 robot_acc
float64 z_tolerance
float64 angle_tolerance
float64 radius
float64 angle_limit
int32 num_pt
float64 z_height
bool autofocus
bool freedrive
bool next
bool previous
bool home
bool reset
bool scan_trigger
bool scan_3d
bool full_scan
bool robot_mode
bool oct_mode
bool octa_mode
bool oce_mode
